static/app/views/performance/http/httpDomainSummaryPage.tsx

```diff
@@ -89,6 +90,16 @@ export function HTTPDomainSummaryPage() {
     referrer: 'api.starfish.http-module-domain-summary-duration-chart',
   });
 
+  const {
+    isLoading: isResponseCodeDataLoading,
+    data: responseCodeData,
+    error: responseCodeError,
+  } = useSpanMetricsSeries({
+    filters: filters,
+    yAxis: ['http_response_rate(3)', 'http_response_rate(4)', 'http_response_rate(5)'],
+    referrer: 'api.starfish.http-module-domain-summary-response-code-chart',
```

Comment:
Don't forget to update the BE with these referrers otherwise they get merged in with the default referrer instead
https://github.com/getsentry/sentry/blob/master/src/sentry/snuba/referrer.py#L390

Reply:
Bah! Thanks for the reminder, I'll do that 👍🏻